Pcap++/src/DpdkDeviceList.cpp

```diff
@@ -87,9 +93,12 @@ bool DpdkDeviceList::initDpdk(CoreMask coreMask, uint32_t mBufPoolSizePerDevice,
 		}
 	}
 
-	if (!verifyHugePagesAndDpdkDriver())
+	if (m_DpdkInitVerify)
```

Comment:
Nitpick but maybe more readable like this no ?

```suggestion
	if (m_DpdkInitVerify && !verifyHugePagesAndDpdkDriver())
		return false;
```

Reply:
good good